spec/fixtures/constants.rb

```diff
@@ -59,6 +59,11 @@
 )
+SAMPLE_SWIFT_COMPILE = %Q(
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.